nntrainer/compiler/bn_realizer.cpp

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2022 Jijoong Moon <jijoong.moon@samsung.com>
+ *
+ * @file bn_realizer.cpp
+ * @date 13 April 2022
+ * @brief NNTrainer graph realizer which remove batch normalization layer for
+ * inference
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+#include <bn_realizer.h>
+#include <connection.h>
+#include <layer_node.h>
+#include <nntrainer_error.h>
+#include <nntrainer_log.h>
+
+#include <algorithm>
+#include <stdexcept>
+#include <unordered_map>
+
+namespace nntrainer {
+
+static constexpr size_t SINGLE_INOUT_IDX = 0;
+
+BnRealizer::BnRealizer() {}
+
+BnRealizer::~BnRealizer() {}
```

Comment:
Just curiosity, is there any reason why these default constructor and destructor are declared even though they are implicitly done?

Reply:
No reason. It's just from the skeleton code. I will remove them in a later pr. Thanks.

Reply:
If it is just convention things, don't bother removing them:) Anyway looks good to me.